runhouse/main.py

```diff
@@ -576,6 +590,94 @@ def status(
     _print_status(cluster_status, current_cluster)
 
 
+async def aget_clusters_from_den():
+
+    get_clusters_params = {"resource_type": "cluster", "folder": rns_client.username}
+    clusters_in_den_resp = rns_client.session.get(
+        f"{rns_client.api_server_url}/resource",
+        params=get_clusters_params,
+        headers=rns_client.request_headers(),
+    )
+
+    return clusters_in_den_resp
+
+
+def get_clusters_from_den():
```

Comment:
if we use the rns sesson client i think we can call it directly without wrapping in asyncio?